ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/signing/fileBased/FileBasedSignerHelper.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.signing.fileBased;
+
+import tech.pegasys.ethsigner.core.signing.CredentialTransactionSigner;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import com.google.common.base.Charsets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.web3j.crypto.CipherException;
+import org.web3j.crypto.Credentials;
+import org.web3j.crypto.WalletUtils;
+
+public class FileBasedSignerHelper {
```

Comment:
This name doesn't seem quite right, is this a factory perhaps?

Reply:
We could make it a Builder? What do you think?

Reply:
Only if you also make it into a `Builder` 😉

Reply:
Done